src/tests/ftest/control/dmg_scale.py

```diff
@@ -0,0 +1,185 @@
+    "engine_pool_ops_obj_punch",
+    "engine_pool_ops_obj_sync",
+    "engine_pool_ops_recx_enum",
+    "engine_pool_ops_tgt_akey_punch",
+    "engine_pool_ops_tgt_dkey_punch",
+    "engine_pool_ops_tgt_punch",
+    "engine_pool_ops_tgt_update",
+    "engine_pool_ops_update",
+    "engine_pool_ops_pool_connect",
+    "engine_pool_ops_pool_disconnect",
+    "engine_pool_ops_pool_evict",
+    "engine_pool_ops_pool_query",
+    "engine_pool_ops_pool_query_space",
+    "engine_pool_resent",
+    "engine_pool_restarted",
+    "engine_pool_retry",
+    "engine_pool_scrubber_busy_time",
+    "engine_pool_scrubber_bytes_scrubbed_current",
+    "engine_pool_scrubber_bytes_scrubbed_prev",
+    "engine_pool_scrubber_bytes_scrubbed_total",
+    "engine_pool_scrubber_corruption_current",
+    "engine_pool_scrubber_corruption_total",
+    "engine_pool_scrubber_csums_current",
+    "engine_pool_scrubber_csums_prev",
+    "engine_pool_scrubber_csums_total",
+    "engine_pool_scrubber_next_csum_scrub",
+    "engine_pool_scrubber_next_tree_scrub",
+    "engine_pool_scrubber_prev_duration",
+    "engine_pool_scrubber_prev_duration_max",
+    "engine_pool_scrubber_prev_duration_mean",
+    "engine_pool_scrubber_prev_duration_min",
+    "engine_pool_scrubber_prev_duration_stddev",
+    "engine_pool_scrubber_scrubber_started",
+    "engine_pool_scrubber_scrubs_completed",
+    "engine_pool_started_at",
+    "engine_pool_vos_aggregation_akey_deleted",
+    "engine_pool_vos_aggregation_akey_scanned",
+    "engine_pool_vos_aggregation_akey_skipped",
+    "engine_pool_vos_aggregation_csum_errors",
+    "engine_pool_vos_aggregation_deleted_ev",
+    "engine_pool_vos_aggregation_deleted_sv",
+    "engine_pool_vos_aggregation_dkey_deleted",
+    "engine_pool_vos_aggregation_dkey_scanned",
+    "engine_pool_vos_aggregation_dkey_skipped",
+    "engine_pool_vos_aggregation_epr_duration",
+    "engine_pool_vos_aggregation_epr_duration_max",
+    "engine_pool_vos_aggregation_epr_duration_mean",
+    "engine_pool_vos_aggregation_epr_duration_min",
+    "engine_pool_vos_aggregation_epr_duration_stddev",
+    "engine_pool_vos_aggregation_merged_recs",
+    "engine_pool_vos_aggregation_merged_size",
+    "engine_pool_vos_aggregation_obj_deleted",
+    "engine_pool_vos_aggregation_obj_scanned",
+    "engine_pool_vos_aggregation_obj_skipped",
+    "engine_pool_vos_aggregation_uncommitted",
+    "engine_pool_vos_space_nvme_used",
+    "engine_pool_vos_space_scm_used",
+    "engine_pool_xferred_fetch",
+    "engine_pool_xferred_update",
+    "engine_pool_EC_update_full_stripe",
+    "engine_pool_EC_update_partial",
+    "engine_pool_block_allocator_alloc_hint",
+    "engine_pool_block_allocator_alloc_large",
+    "engine_pool_block_allocator_alloc_small",
+    "engine_pool_block_allocator_frags_aging",
+    "engine_pool_block_allocator_frags_large",
+    "engine_pool_block_allocator_frags_small",
+    "engine_pool_block_allocator_free_blks",
+    "engine_pool_ops_key2anchor"
+]
```

Comment:
Any reason why we want to keep this here? I feel we have to use the metrics list available under TelemetryUtils.py.

Reply:
I think it's better to keep them here because these are scattered across different variables in TelemetryUtils.py. Also, they can be moved around or removed by someone else in TelemetryUtils.py.

Reply:
FWIW @phender And I have gone back and forth on this too. I tend to agree with @shimizukko here: keeping them here makes it much less likely that someone accidentally breaks them in the utils